src/components/root_layout.tsx

```diff
@@ -18,7 +18,7 @@ interface RootLayoutProps {
  */
 const RootLayout: FC<RootLayoutProps> = ({children}) => {
   return (
-    <div style={{marginLeft: "auto", marginRight: "auto", maxWidth: 800}}>
+    <div style={{marginLeft: "auto", marginRight: "auto", maxWidth: 800, backgroundColor: "rgba(211, 211, 211, 0.5)"}}>
```

Comment:
I wonder if it would be slightly better to set the background color to rgb(105, 105, 105). Setting the transparency to 0.5 to get a lighter shade of grey assumes that the underlying background is white. Setting the rgb values explicitly eliminates that possible dependency.

After May 8, we should have a discussion regarding the best way to style our components. But this is a quick fix that is just what the doctor ordered.